server/src/tasks/sqs/upgrade.js

```diff
@@ -0,0 +1,29 @@
+const config = require('config')
+const { merge, omit } = require('lodash')
+const { createNetwork } = require('@utils/web3')
+const mongoose = require('mongoose')
+const { sendRelay, isAllowedToRelay } = require('@utils/relay')
+const UserWallet = mongoose.model('UserWallet')
+const WalletUpgrade = mongoose.model('WalletUpgrade')
+
+const installUpgrade = async (account, { walletAddress, methodName, methodData, nonce, gasPrice, gasLimit, signature, walletModule, network, upgradeId }, job) => {
```

Comment:
I did a simpler solution for now.
But I think that's a good refactor also, It's not making much sense putting all the relays  into one endpoint